cmd/parca-agent/main.go

```diff
@@ -170,6 +182,11 @@ func main() {
 		}
 	}
 
+	if flags.CPUSamplingFrequency <= 0 {
+		level.Warn(logger).Log("msg", "cpu sampling frequency is too low. Setting it to the default value", "default", defaultCPUSamplingFrequency)
+		flags.CPUSamplingFrequency = defaultCPUSamplingFrequency
+	}
```

Comment:
Perhaps we could also have a warning if the frequency is bigger than some threshold, e.g. >150Hz?

Reply:
Sure 👍